packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/MultiWorkspaceDropdown/internal/MultiWorkspaceDropdownDefaultComponents.tsx

```diff
@@ -0,0 +1,174 @@
+          AppPath.Verify,
+          {
+            loginToken: data.signUpInNewWorkspace.loginToken.token,
+          },
+          '_blank',
+        );
+      },
+      onError: (error: Error) => {
+        enqueueSnackBar(error.message, {
+          variant: SnackBarVariant.Error,
+        });
+      },
+    });
+  };
+
+  return (
+    <DropdownMenuItemsContainer>
+      <DropdownMenuHeader
+        StartAvatar={
+          <Avatar
+            placeholder={currentWorkspace?.displayName || ''}
+            avatarUrl={currentWorkspace?.logo ?? DEFAULT_WORKSPACE_LOGO}
+          />
+        }
+        DropdownOnEndIcon={
+          <Dropdown
+            clickableComponent={
+              <LightIconButton
+                Icon={IconDotsVertical}
+                size="small"
+                accent="tertiary"
+              />
+            }
+            dropdownId={'multi-workspace-dropdown-context-menu'}
+            dropdownHotkeyScope={{ scope: SelectHotkeyScope.Select }}
+            dropdownComponents={
+              <DropdownMenuItemsContainer>
+                <MenuItem
+                  LeftIcon={IconPlus}
+                  text={t`Create Workspace`}
+                  onClick={createWorkspace}
+                />
+              </DropdownMenuItemsContainer>
+            }
+          />
+        }
+      >
+        {currentWorkspace?.displayName}
+      </DropdownMenuHeader>
+      {workspaces
+        .filter(({ id }) => id !== currentWorkspace?.id)
+        .slice(0, 3)
+        .map((workspace) => (
+          <UndecoratedLink
+            key={workspace.id}
+            to={buildWorkspaceUrl(getWorkspaceUrl(workspace.workspaceUrls))}
+            onClick={(event) => {
+              event?.preventDefault();
+              handleChange(workspace);
+            }}
+          >
+            <MenuItemSelectAvatar
+              text={workspace.displayName ?? '(No name)'}
+              avatar={
+                <Avatar
+                  placeholder={workspace.displayName || ''}
+                  avatarUrl={workspace.logo ?? DEFAULT_WORKSPACE_LOGO}
+                />
+              }
+              selected={currentWorkspace?.id === workspace.id}
+            />
+          </UndecoratedLink>
+        ))}
+      {workspaces.length > 4 && (
+        <MenuItem
+          LeftIcon={IconSwitchHorizontal}
+          text={t`Other workspaces`}
+          onClick={() => setMultiWorkspaceDropdownState('workspaces-list')}
+          hasSubMenu={true}
+        />
+      )}
+      <DropdownMenuSeparator />
+      <MenuItem
+        LeftIcon={colorSchemeList.find(({ id }) => id === colorScheme)?.icon}
+        text={
+          <>
+            {t`Theme `}
+            <StyledDescription>{` · ${colorScheme}`}</StyledDescription>
+          </>
+        }
+        hasSubMenu={true}
+        onClick={() => setMultiWorkspaceDropdownState('themes')}
+      />
+      <NavigationDrawerItem
+        label={t`Invite user`}
```

Comment:
Strange bug where invite user sometimes appear as bold? Not sure why!

<img width="251" alt="Screenshot 2025-03-12 at 11 03 07" src="https://github.com/user-attachments/assets/af2781bc-12c9-457d-8849-0cf0fd441cf4" />